pkg/sql/logictest/tests/local-mixed-23.1/generated_test.go

```diff
@@ -1184,6 +1184,13 @@ func TestLogic_merge_join(
 	runLogicTest(t, "merge_join")
```

Comment:
23.1 mixed? this is a new feature in 24.1 and we don't want to test it in mixed version mode do we?

Reply:
This is gone now.